bin/pytorchMPSevolve.py

```diff
@@ -0,0 +1,144 @@
+import numpy as np
+import torch
+from torch.autograd import Variable
+
+def init_spinup_MPS(N, D=1):
+    """MPS: all spins up
+    Args
+    ----
+    D : physical dimension, product state for D=1
+    """
+    Ms = np.zeros([N, D, 2, D], dtype=np.csingle)
```

Comment:
```suggestion
    Ms = torch.zeros([N, D, 2, D], dtype=torch.float32)
```

Then `Ms.requires_grad = True` to make it differentiable.